modules/xfmg/json/XMOM/xfmg/xfctrl/datamodel/json/parameter/JSONWritingOptions.xml

```diff
@@ -37,4 +37,10 @@ If not set or set to a different value no membervars will be skipped.</Documenta
     </Meta>
   </Data>
   <Data Label="List Wrapper" VariableName="listWrapper" IsList="true" ReferenceName="ListWrapper" ReferencePath="xfmg.xfctrl.datamodel.json.parameter" />
+  <Data Label="Process All Input Generically" VariableName="processAllInputGenerically">
+    <Meta>
+      <Type>Boolean</Type>
+      <Documentation>If set to true, xfmg.xfctrl.datamodel.json.JSONObject and xfmg.xfctrl.datamodel.json.JSONValue are written using the generic serialization process. If set to null, xfmg.json.GenericProcessing is used!</Documentation>
```

Comment:
Add to the documentation (here and in JSONParsingOptions):

1. xfmg.json.GenericProcessing is a XynaProperty
2. Explain what happens for "false"